schema/objects/StockPlan.schema.json

```diff
@@ -34,13 +34,21 @@
       "$ref": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/enums/StockPlanCancellationBehaviorType.schema.json"
     },
     "stock_class_id": {
-      "description": "Identifier of the StockClass object this plan is composed of",
-      "type": "string"
+      "description": "[DEPRECATED in favor of stock_class_ids] Identifier of the StockClass object this plan is composed of.",
+      "type": "string",
+      "deprecated": "true"
+    },
+    "stock_class_ids": {
+      "description": "Identifiers of StockClass objects this plan is composed of",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
     },
     "id": {},
     "comments": {}
   },
   "additionalProperties": false,
-  "required": ["plan_name", "stock_class_id", "initial_shares_reserved"],
+  "required": ["plan_name", "stock_class_ids", "initial_shares_reserved"],
```

Comment:
Making `stock_class_id` optional is fine, but requiring `stock_class_ids` is a breaking change. We'll need to use a `oneOf` construct to require one or the other within 1.x

@arthur-clara validation thought: As we introduce things like this with deprecation notes, it would be awesome to include deprecation warnings in validator output.

Reply:
@lavens and @jarrettq, here's an example of what @pjohnmeyer is referring to:

https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/blob/7abfda1d5332e816c09b387db95a5f57afbafbb7/schema/objects/transactions/issuance/EquityCompensationIssuance.schema.json#L151-L158

Reply:
@pjohnmeyer, it'd also be great to pick up deprecation notices in a central doc and also give them more prominent billing in the schema-specific docs

Reply:
> Making `stock_class_id` optional is fine, but requiring `stock_class_ids` is a breaking change. We'll need to use a `oneOf` construct to require one or the other within 1.x

Good catch, made the update as requested